src/romaine/core.py

```diff
@@ -1,6 +1,45 @@
+import os
+
+
 class Core(object):
+    """
+        The core of the Romaine, provides BDD test API.
+    """
+    # All located features
+    features = []
```

Comment:
Probably should be a set, yes- though it won't fully avoid duplicates (if one set are called with relative paths and one with absolute they'll be different), but that is the sort of edge case that probably introduces more complexity than the pain it saves. Especially as it'd take a pretty odd user approach to cause that sort of duplication.